apps/Gruntfile.js

```diff
@@ -404,6 +404,8 @@ module.exports = function (grunt) {
 
     pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx',
 
+    teacherApplication: './src/code-studio/pd/teacher_application/teacher_application.jsx',
```

Comment:
@pcardune That makes good sense. I'll make a note to reorganize the workshop_dashboard this way as well. 

One question: How then do we pass data from rails to React? Currently I'm populating an object on `window` in the [view](https://github.com/code-dot-org/code-dot-org/blob/bf96a7b6973ef4b81f60d70a6f9f2f0585cadbe8/dashboard/app/views/pd/workshop_dashboard/index.html.haml#L8-L13), and reading it in javascript, e.g. [here](https://github.com/code-dot-org/code-dot-org/blob/30bd63533b1db213afbdfe37eef4466939b5b385/apps/src/code-studio/pd/workshop_dashboard/workshop_index.jsx#L32-L32).